src/Reveal.Sdk.Dom.Tests/Data/DataSourceItems/BoxDataSourceItemFixture.cs

```diff
@@ -0,0 +1,57 @@
+using Reveal.Sdk.Dom.Core.Extensions;
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSourceItems
+{
+    public class BoxDataSourceItemFixture
+    {
+        [Fact]
+        public void Constructor_SetsTitleAndDataSource_ValidDataSource()
```

Comment:
I understand the thinking but I think it's a bit confusing when read the function's name at first. I suggest change it to:

```suggestion
public void Constructor_SetsTitleAndDataSource_WhenConstructed()
```

Basically add `When...`, `With...` something to make it a bit more readable. Apply for the rest pls